### DISCUSSION TITLE
How to script power modes?

### DISCUSSION THREAD

**Discussion by hinchyman** [BartoszCichecki/LenovoLegionToolkit#928] [Q&A]
Hello. I'm using a Lenovo Slim Pro X which isn't fully compatible with Legion Toolkit but I'm wondering how does Toolkit manage Power Modes? I'm trying to script a way to alter (Fn + Q) depending on what I'm doing. If you could give me some pointers or put me in the right direction, I'd really really appreciate it

**Comment by BartoszCichecki** [comment]
I don't know tbh what do you need. You probably need to reverse engineer Vantage for that and find out how Fn+Q works on Slim Pro X.
